etc/kayobe/kolla/config/fluentd/filter/02-audit.conf

```diff
@@ -0,0 +1,5 @@
+<match openstack_python>
+    @type rewrite_tag_filter
+    capitalize_regex_backreference yes
+    rewriterule1 python_module ^oslo.messaging.notification.+$ openstack_audit
```

Comment:
This should work for any openstack service, although currently only keystone is logging notifications